### ISSUE TITLE
Reading: XL-LEXEME: WiC Pretrained Model for Cross-Lingual LEXical sEMantic changE

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#273]
## 0. Paper
 - paper: [[link](https://aclanthology.org/2023.acl-short.135/)]
 - ACL2023

## 1. What is it?
They conduct the semantic change detection tasks with word-in-context tasks.

## 2. What is amazing compared to previous works?
 - Their method achieves SoTA performance in three languages
 - They release pre-trained models in huggingface https://huggingface.co/pierluigic/xl-lexeme

## 3. Where is the key to technologies and techniques?
They train their model (Sentence-BERT + Siamese Network) in word-in-context dataset using contrastive loss.

![スクリーンショット 2023-07-18 5 52 04](https://github.com/a1da4/paper-survey/assets/45454055/ee6d37c3-b31c-4cd5-ac31-b9fc1c330bc4)

In the prediction, they adapt pairwise cosine similarity.

## 4. How did evaluate it?
![スクリーンショット 2023-07-18 5 53 02](https://github.com/a1da4/paper-survey/assets/45454055/d81996e2-019d-439e-a055-9ff3dacd5285)

Their model achieves state-of-the-art performance in three languages.
